libraries/BLE/examples/BLE_Beacon_Scanner/BLE_Beacon_Scanner.ino

```diff
@@ -99,30 +99,17 @@ class MyAdvertisedDeviceCallbacks : public BLEAdvertisedDeviceCallbacks
           Serial.printf("TX power %d\n", foundEddyURL.getPower());
           Serial.println("\n");
         }
-        else if (payLoad[11] == 0x20)
```

Comment:
I just changed Line 102  to:
`else if (payLoad[22] == 0x20)`

in order to make it work.